### ISSUE TITLE
Sonoff RF gives 2 events per action

### ISSUE THREAD

**Issue by miketree1** [AlexxIT/SonoffLAN#126]
I have just upgraded to 2.0.3
Now for each action (ie ring doorbell) I get 2 events.
I removed existing custom_components/sonoff before installing via hacs.

```
{
    "event_type": "sonoff.remote",
    "data": {
        "entity_id": "remote.sonoff_10007c517b",
        "command": 1,
        "ts": "2020-05-27T13:08:50.000Z",
        "name": "bell"
    },
    "origin": "LOCAL",
    "time_fired": "2020-05-27T13:08:51.189386+00:00",
    "context": {
        "id": "18d361da2aca45b68ecea9f72ad1a28b",
        "parent_id": null,
        "user_id": null
    }
}

Event 2 fired 2:08 PM:

{
    "event_type": "sonoff.remote",
    "data": {
        "entity_id": "remote.sonoff_10007c517b",
        "command": 1,
        "ts": "2020-05-27T13:08:50.000Z",
        "name": "bell"
    },
    "origin": "LOCAL",
    "time_fired": "2020-05-27T13:08:51.153641+00:00",
    "context": {
        "id": "a9a00705c36b4b27aff61facee86f3fc",
        "parent_id": null,
        "user_id": null
    }
}
```

**Comment by AlexxIT** [comment]
Please show logs https://youtu.be/Lt5fT4N5Pm8

**Comment by miketree1** [comment]
I've sorted it. I noticed in the log a cloud event and a local event.
I've set   mode: local & now only one event.
Thanks
:)

**Comment by AlexxIT** [comment]
This is bad. Duplicate messages should be filtered. I will check it

**Comment by miketree1** [comment]
In that case, do you still want the log?

**Comment by AlexxIT** [comment]
I have a Bridge. Can do it myself.

**Comment by AlexxIT** [comment]
Fixed in 2.0.4
